R/auth.R

```diff
@@ -329,3 +323,13 @@ twitter_init_oauth1.0 <- function (endpoint, app, permission = NULL,
     private_key = private_key
   )
 }
+
+auth_path <- function(...) {
+  # Use private option to make testing easier
+  if (getRversion() >= "4.0.0") {
```

Comment:
This conditioning means that different directories are used for different versions of R, e.g.:

``` r
tools::R_user_dir("rtweet", "config")
#> [1] "~/Library/Preferences/org.R-project.R/R/rtweet"
rappdirs::user_config_dir("rtweet")
#> [1] "~/Library/Application Support/rtweet"
```

This means that upgrading R might lose your token, and it'll be abandoned in a directory that is easy to forget about it. I think you need to pick one of the two options, rather than trying to support both.

Reply:
Then you need to delete this code branch.